certified-connectors/CyberProof/apiProperties.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "cp_instance": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "CyberProof URL",
+          "description": "CyberProof URL (must support https). ",
+          "tooltip": "Provide your CyberProof URL",
+          "constraints": {
+            "required": "true"
+          }
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "CybeProof API key",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#f9ae0c",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamicHostUrl",
+        "title": "Set host from connection parameter",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('cp_instance')"
+        }
+      },
+      {
+        "templateId": "setheader",
+        "title": "Set authorization header",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "St2-Api-Key",
+          "x-ms-apimTemplateParameter.value": "@connectionParameters('api_key')",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      }
+    ],
+    "publisher": "CyberProof Inc.",
+    "stackOwner": "CyberProof Inc."
```

Comment:
@DavidMeu You have provided stack owner as "CyberProof Inc." and submitted privacy policy from UST.com https://www.ust.com/en/legal-notice/privacy-policy.

Can you please let me know if "CyberProof Inc." is a fully qualified organization to publish this connector.

Reply:
Yes it is.
"CyberProof Inc." is a subsidiary of UST